.github/workflows/operate-release-reusable.yml

```diff
@@ -146,33 +137,51 @@ jobs:
 
       #########################################################################
       # Release: run Maven release
-      - name: Maven release
+      - name: Prepare Maven release
         env:
-          PUSH_CHANGES: ${{ inputs.dryRun == false }}
-          SKIP_DEPLOY: ${{ inputs.dryRun }}
           NEXT_DEVELOPMENT_VERSION: ${{ inputs.nextDevelopmentVersion }}
         run: |
-          mvn -f operate release:prepare release:perform -P -docker \
-          -DpushChanges=$PUSH_CHANGES \
-          -DlocalCheckout=true \
+          mvn -f operate release:prepare -P -docker \
+          -DpushChanges=false \
           -DskipTests=true -B -T$LIMITS_CPU --fail-at-end \
           -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn \
           -Dtag=$TAG_VERSION \
           -DreleaseVersion=$RELEASE_VERSION \
           -DdevelopmentVersion=$NEXT_DEVELOPMENT_VERSION \
-          -Darguments='-Dmaven.deploy.skip=$SKIP_DEPLOY -P -docker -DskipTests=true -DskipNexusStagingDeployMojo=$SKIP_DEPLOY -B --fail-at-end -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -f pom.xml'
+          -Darguments='-Dmaven.deploy.skip=$DRY_RUN -P -docker -DskipTests=true -DskipNexusStagingDeployMojo=$DRY_RUN -B --fail-at-end -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -f pom.xml'
 
-      - name: Create a release
+      # Pushing changes manually since maven-release-plugin fails in subfolders:
+      # https://issues.apache.org/jira/browse/MRELEASE-885
+      - name: Push Git changes
+        if: ${{ env.CREATE_A_RELEASE == 'true' }}
+        run: |
+          git push --tags
+          git push --all
+
+      - name: Perform Maven release
+        run: |
+          mvn -f operate release:perform -P -docker \
+          -DlocalCheckout=true \
+          -DskipTests=true -B -T$LIMITS_CPU --fail-at-end \
+          -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn \
+          -Darguments='-Dmaven.deploy.skip=$DRY_RUN -P -docker -DskipTests=true -DskipNexusStagingDeployMojo=$DRY_RUN -B --fail-at-end -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -f pom.xml'
+
+      - name: Create a GitHub release
         if: ${{ env.CREATE_A_RELEASE == 'true' }}
         env:
           GITHUB_TOKEN: ${{ steps.secrets.outputs.GITHUB_TOKEN }}
         uses: octokit/request-action@v2.1.9
         with:
           route: POST /repos/camunda/zeebe/releases
-          draft: false
+          draft: true
           name: ${{ env.TAG_VERSION }}
           tag_name: ${{ env.TAG_VERSION }}
-          body: ${{ toJSON(format('{0}', steps.changelog.outputs.body)) }}
+          generate_release_notes: true
```

Comment:
what is the effect of `generate_release_notes : true` ? what did you get when you did the test?

Reply:
No verbose release notes (not sure why), GitHub instead inserted a link to the commit range between tag 8.4.5 and tag operate-8.5.0-test1 - I think it also got confused of the merge history.

Reply:
I'd still keep the option since it doesn't hurt and the release manager can choose to overwrite.

Reply:
API docs: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release